jipdate.py

```diff
@@ -191,7 +196,7 @@ def get_jira_issues(jira, exclude_stories, epics_only, all_status, filename,
         f.write("# Header: %s\n" % issue.fields.summary)
         f.write("# Type: %s\n" % issue.fields.issuetype)
         f.write("# Status: %s\n" % issue.fields.status)
-        f.write("No updates since last week.\n\n")
+        f.write("# No updates since last week.\n\n")
```

Comment:
IMHO, "No update since last week" is a bit useless and will just turn out to have some 'in progress' cards with dozen of  "No update since last week" that will be seen as a card that is in progress and that moves forward because there are regular comments but that is in fact stalled.
Never mind, as long as we can configure the tools to not add such default comment if wanted, i'm fine.